Sources/Brave/Frontend/Browser/Toolbars/BottomToolbar/Menu/Bookmarks/BookmarksViewController.swift

```diff
@@ -443,16 +443,26 @@ class BookmarksViewController: SiteTableViewController, ToolbarUrlActionsProtoco
     let fontSize: CGFloat = 14.0
     cell.textLabel?.text = item.title ?? item.url
     cell.textLabel?.lineBreakMode = .byTruncatingTail
+    cell.detailTextLabel?.lineBreakMode = .byTruncatingTail
 
     if !item.isFolder {
       configCell()
       cell.textLabel?.font = UIFont.systemFont(ofSize: fontSize)
       cell.accessoryType = .none
+      cell.detailTextLabel?.text = nil
     } else {
       configCell(image: UIImage(braveSystemNamed: "leo.folder"))
       cell.textLabel?.font = UIFont.boldSystemFont(ofSize: fontSize)
       cell.accessoryType = .disclosureIndicator
       cell.setRightBadge(nil)
+      
+      // 0 bookmarks
+      // 1 bookmark
+      // 2 bookmarks
+      // ...
+      cell.detailTextLabel?.text = String(format: item.bookmarkNode.childCount == 1 ? Strings.bookmarkSingleCount :
```

Comment:
Bookmark node childCount includes subfolders right ? So this will give us a number showing the number of bookmarks + folder for that particular folder.

I think this might be confusing maybe we should specifically show user, their bookmark count and folder count separately.

Like `5 Bookmarks - 3 Folders`

Hmm what do you think ?

Reply:
Oops. This is supposed to be `children.count` like in the line below it. Will fix it.